Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_22.md

```diff
@@ -0,0 +1,27 @@
+#### Incident Fields
+
+##### ASM - Remediation Objectives
+
+Added the **ASM - Remediation Objectives** field used to remediation actions.
+
+#### Playbooks
+
+##### New: Cortex ASM - Remediation Objectives
+
+Added the "Cortex ASM - Remediation Objectives" Playbook that populates remediation objectives field that is used to display the remediation actions to the end user (Available from Cortex XSOAR 6.8.0).
+
+##### Cortex ASM - Remediation Path Rules
+
+Updated the playbook to incorporate the new "Cortex ASM - Remediation Objectives" sub-playbook.
+
+##### Cortex ASM - Prisma Cloud Enrichment
+
+Updated the playbook to skip sub-playbooks of separate packs if unavailable.
+
+##### Cortex ASM - Remediation
+
+Updated the playbook to skip sub-playbooks of separate packs if unavailable.
```

Comment:
This description is the same as the previous one.  Is that correct?

Reply:
yes